blt/urls.py

```diff
@@ -415,7 +415,8 @@
     re_path(r"^contributors/$", contributors_view, name="contributors"),
     path("company/", include("company.urls")),
     path("sponsor/", website.views.sponsor_view, name="sponsor"),
-    path("companies/", DomainListView.as_view(), name="domain_lists"),
+    path("companies/", DomainListView.as_view(), name="domain_list"),
```

Comment:
Why did you change this?

Reply:
Sorry! Changed it , Everything is working perfectly now